tests/model_serving/model_server/private_endpoint/conftest.py

```diff
@@ -0,0 +1,189 @@
+import json
+import pytest
+from typing import Generator, Any
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.secret import Secret
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from simple_logger.logger import get_logger
+from ocp_resources.serving_runtime import ServingRuntime
+from kubernetes.dynamic import DynamicClient
+
+from utilities.serving_runtime import ServingRuntimeFromTemplate
+from tests.model_serving.model_server.storage.pvc.utils import create_isvc
+from tests.model_serving.model_server.private_endpoint.utils import (
+    create_sidecar_pod,
+    get_flan_deployment,
+    b64_encoded_string,
+)
+from tests.model_serving.model_server.private_endpoint.infra import create_ns
```

Comment:
we have `from tests.utils import create_ns` 

please move `create_ns` to utilities.infra.py and use in both places